README.md

```diff
@@ -20,10 +20,19 @@ yarn add @noamalffasy/js-whatsapp
 
 ### Setting up
 
+All parameters are optional.  The following are the default values.  
 ```js
 import Whatsapp from "@noamalffasy/js-whatsapp";
 
-const wap = new Whatsapp();
+const params = {
+  keysPath : "./keys.json",
+  qrPath : "./qrcode.png",
+  clientName : "WhatsApp forwarder",
+  clientShortName : "WhatsAppForwarder",
+  restoreSession : false
+}
+
+const wap = new Whatsapp(params);
```

Comment:
```suggestion
const opts = {
  keysPath: "./keys.json",
  qrPath: "./qrcode.png",
  clientName: "WhatsApp forwarder",
  clientShortName: "WhatsAppForwarder",
  restoreSession: false
}

const wap = new Whatsapp(opts);
```
I'd change the name to this instead, just a preference.
I also changed the code style (no space before the` :` and after the property name).